app/src/main/java/ch/epfl/sweng/SDP/utils/RankingUtils.java

```diff
@@ -21,12 +22,17 @@ private RankingUtils() {
      */
     public static Integer[] generateTrophiesFromRanking(Integer[] ranking) {
         Integer[] trophies = new Integer[ranking.length];
-        trophies[0] = MAX_RANK;
+
```

Comment:
Why did you change this method? (Substitute algorithm)

Reply:
The method was improved